GameSync.Api.UnitTests/Controllers/AccountControllerTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Xunit;
```

Comment:
Missing integration test that would check if user was saved in database after successful request